### ISSUE TITLE
Questions about the inference

### ISSUE THREAD

**Issue by slcheng97** [orrzohar/PROB#3]
Hi, thanks for your great work!
In the inference phase, I find that your model may assign multi-class labels to the same token in the inference phase, is this reasonable?

**Comment by orrzohar** [comment]
Hi @chengsilin,
Thank you! Yes, this indeed happens, and it follows both [Deformable DETR](https://github.com/fundamentalvision/Deformable-DETR/blob/11169a60c33333af00a4849f1808023eba96a931/models/deformable_detr.py#L412) and [OW-DETR](https://github.com/akshitac8/OW-DETR/blob/af7d66618f3842ebe2e25720fbca6028a8701905/models/deformable_detr.py#L631). 

Briefly, the convention was used by D-DETR when they removed the 'background' logit and opted to use a sigmoid rather than a softmax in inference. The VOC evaluation protocol allows for this type of inference, where the predictions are rank-ordered in terms of confidence before counting the correct/incorrect predictions. For a fair comparison, I opted not to change the inference, leaving the number of predictions per image at 100.

If you have any other questions, I'd be happy to answer them.
Orr
